pkg/cmd/repo/view/http.go

```diff
@@ -15,13 +15,15 @@ var NotFoundError = errors.New("not found")
 type RepoReadme struct {
 	Filename string
 	Content  string
+	BaseURL  string
 }
 
 func RepositoryReadme(client *http.Client, repo ghrepo.Interface, branch string) (*RepoReadme, error) {
 	apiClient := api.NewClientFromHTTP(client)
 	var response struct {
-		Name    string
-		Content string
+		Name     string
+		Content  string
+		Html_url string
```

Comment:
Snake case is not a Go convention, maybe change it to HtmlURL??

Reply:
Nice catch, [TIL](https://github.com/golang/go/wiki/CodeReviewComments#initialisms) 😄 .